nix/pkgs/openapi-generator/default.nix

```diff
@@ -0,0 +1,68 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, maven, jdk, jre, makeWrapper }:
```

Comment:
can we instead symlink ./nix/pkgs/openapi-generator from call-home/mayastor-control-plane/nix/pkgs/openapi-generator ? So any updates can simply be picked up by updating the submodule.